cmd/server/main.go

```diff
@@ -78,3 +81,9 @@ func instrumentationSetup() {
 		}
 	}()
 }
+
+func pprofSetup() {
```

Comment:
Enabled pprof. I wanted to inspect some things about a memprofile of the service. 
All good. Left it here anyways, is binded to `localhost` so not public.